doc/changes/latest.inc

```diff
@@ -15,6 +15,8 @@ Current (0.20.dev0)
 Changelog
 ~~~~~~~~~
 
+- Improved :func:`mne.plot_evokeds` to label epoch counts starting from 0 `Sophie Herbst`_
```

Comment:
You have to refer to these by where they show up in the MNE API docs: https://mne.tools/dev/generated/mne.viz.plot_evoked.html

Also does it change `mne.viz.plot_evoked` or `mne.viz.plot_epochs`?
```suggestion
- Improved :func:`mne.viz.plot_epochs` to label epoch counts starting from 0 `Sophie Herbst`_
```

Reply:
Ups, thank you @larsoner . Done.